### ISSUE TITLE
Package conversion

### ISSUE THREAD

**Issue by HemanthSai7** [Math-behind-AI/ScratchAI#14]
**Is your feature request related to a problem? Please describe.**
It is not actually a problem but just a suggestion. Currently, we use the common functions from the sklearn library.

**Describe the solution you'd like**
What if we create a **utils** directory and store all the common loss functions, metrics, etc so that by calling the desired function we can use them?

**Additional context**
For example in the utils directory, it can have a file named loss functions and it can have functions such as cross-entropy loss, etc. And in future examples, instead of using the loss function from another library, we can directly use it which has already been implemented by the community.
